apps/web/src/editor.css

```
@@ -141,138 +230,123 @@
+    transform 140ms ease;
 }
 
 .editor-toc-rail__tick {
   display: block;
-  min-width: 0.7rem;
+  width: var(--toc-tick-width, 0.75rem);
+  min-width: 0.5rem;
   height: 1px;
-  background: color-mix(in oklch, var(--foreground) 42%, transparent 58%);
+  margin-block: auto;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)